src/main/java/formflow/library/ScreenController.java

```diff
@@ -520,6 +574,17 @@ RedirectView postSubflowScreen(
 
     handleAddressValidation(submission, formSubmission);
 
+    Optional<SubflowRelationship> subflowRelationship = subflowManager.subflowRelationship(flow, currentScreen.getSubflow());
+    if (subflowRelationship.isPresent() && subflowRelationship.get().getRepeatFor() != null) {
+      RepeatFor repeatFor = subflowRelationship.get().getRepeatFor();
+      String inputNameKey = repeatFor.getInputName();
```

Comment:
We should probably null check the input name and return a useful error here as it's possible they may have typo'd in their YAML configuration and not have realized it.